config/ols-rbac.yaml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
split up the roles/rbac that the kube-rbac proxy itself needs (to do subject access reviews) from the role/rbac we'd grant a user to let them invoke OLS

Reply:
Makes sense. Changed as suggested.